src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -36,10 +36,10 @@ const processOptimoBlocks = pipe(
   addSummaryBlock,
   augmentWithTimestamp,
   addBylineBlock,
+  addRecommendationsBlock,
```

Comment:
What is the significance of this change?

Reply:
this just ensures that the `addIdsToBlocks` rule gets applied after the recommendations block gets added as it'ss rendered by the `Blocks` container which requires a uuid `id` prop